src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -30,8 +30,9 @@ class JsonAdaptedPerson {
     private final String phone;
     private final String email;
     private final String address;
-
+    private final String organisation;
     private String priority;
+
```

Comment:
@nigellqx I am not sure whether the field priority should be final as well?

Reply:
I agree with you, I think it should be final as well.